parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -305,6 +305,20 @@ else
         # -________________-
         systemctl restart --no-block apt-daily.service
         aptmarkWALinuxAgent unhold &
+    elif [[ $OS == $MARINER_OS_NAME ]]; then
+        {{- if EnableUnattendedUpgrade }}
+        # By default the dnf-automatic is service is notify only in Mariner.
+        # Enable the automatic install timer and the check-restart timer.
+        # Stop the notify only dnf timer since we've enabled the auto install one.
+        # systemctlDisableAndStop adds .service to the end which doesn't work on timers.
+        systemctl disable dnf-automatic-notifyonly.timer
```

Comment:
curious how it was notifying before. just writing out a messag to some file?

Reply:
By default it prints to `/etc/motd`, alternatively it can also e-mail, or print to stdio